backend/src/api/database-migration.ts

```diff
@@ -90,6 +91,29 @@ class DatabaseMigration {
         await this.$executeQuery(connection, 'DROP table IF EXISTS blocks;');
         await this.$executeQuery(connection, this.getCreateBlocksTableQuery(), await this.$checkIfTableExists('blocks'));
       }
+      if (databaseSchemaVersion < 5 && isBitcoin === true) {
```

Comment:
Note that I moved this query here because it's an `ALTER` and therefore cannot be part of a `TRANSACTION`